cli/command/command.go

```diff
@@ -89,10 +89,23 @@ func ConvertCommand() cli.Command {
 func UpCommand() cli.Command {
 	return cli.Command{
 		Name:  "up",
-		Usage: "Submit rc, svc objects to kubernetes API endpoint",
+		Usage: "Deploy your Dockerized application to Kubernetes (by default create dc and svc)",
```

Comment:
`dc` isn't an official abbreviation for deployment. Use Deployments and Services (since most kompose users wouldn't be familiar with k8s abbreviations and it's hard to Google for "what svc means").